fe/src/main/java/org/apache/doris/analysis/RangePartitionDesc.java

```diff
@@ -145,9 +149,14 @@ public PartitionInfo toPartitionInfo(List<Column> schema, Map<String, Long> part
             boolean find = false;
             for (Column column : schema) {
                 if (column.getName().equalsIgnoreCase(colName)) {
-                    if (!column.isKey()) {
+                    if (!column.isKey() && column.getAggregationType() != AggregateType.NONE) {
```

Comment:
Why does it need a second check here?

Reply:
Historical issues...
It is OK, may be removed in future.